### ISSUE TITLE
Sea state clustering

### ISSUE THREAD

**Issue by Amerlon** [SNL-WaterPower/WecOptTool#40]
I believe both the Sorting_and_Clustering_v2.m and the Bretschneider_weapper.m are up and fully working and they need to be integrated into the rest of the optimization code, The code outputs 8 different sea states and I am unaware if the rest of the code can intake more then one.

**Comment by ryancoe** [comment]
@ssolson - Please move this to [MHKit](https://github.com/MHKiT-Software/MHKiT-Python) and tackle it there.

**Comment by ssolson** [comment]
Ryan I tried this but the repositories must be in the same organization to do this. For the GitHub Documentation:
```
You can only transfer issues between repositories owned by the same user or organization account.
```
In this case, MHKiT is an organization and SNL-Waterpower is another. I just got off the phone with Aubrey discussing using the modified IFROM method to calculate probabilities at points. I have created a Gaussian Mixture Model, and a k-means method of doing this. It is my plan to submit a draft PR to MHKiT this week and reference this issue in it. I will close this issue at that time.
